Automattic/jetpack
# E2E tests: log Jetpack version

#### Changes proposed in this Pull Request:

Use the REST API to get the list of plugins and log them into environment.json. The Jetpack version is further logged in the test report and in the Slack notification.

Note that for local sites using Docker this doesn't work just yet, because an application password needed to authenticate the API is not available. I'd rather postpone handling this to a future PR, as the need of logging the Jetpack version was for atomic sites.

#### Jetpack product discussion
1156386383419466-as-1200924739850039

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Run the tests against a non-local site and check the Slack notification and the test report (for each test in the overview section). For local sites, the logged version will be "unknown".
